usr/src/cmd/zdb/zdb.c

```diff
@@ -4430,6 +4499,161 @@ verify_checkpoint(spa_t *spa)
 	return (error);
 }
 
+/* ARGSUSED */
+static void
+mos_leaks_cb(void *arg, uint64_t start, uint64_t size)
+{
+	for (uint64_t i = start; i < size; i++) {
+		(void) printf("MOS object %llu referenced but not allocated\n",
+		    (long long)i);
```

Comment:
nit: `u_longlong_t` is used for this throughout `zdb` to match the format specifier.